test/e2e/framework/config.go

```diff
@@ -26,6 +26,7 @@ type Config struct {
 	PodIdentityUserAssignedIdentityID string `envconfig:"POD_IDENTITY_USER_ASSIGN_IDENTITY_ID"`
 	ResourceGroup                     string `envconfig:"RESOURCE_GROUP"`
 	IsUpgradeTest                     bool   `envconfig:"IS_UPGRADE_TEST"`
+	IsHelmTest                        bool   `envconfig:"IS_HELM_TEST" default:"true"`
```

Comment:
Could you add this to DeepCopy method too? `copy.IsHelmTest = c.IsHelmTest`

Reply:
done